packages/mobile/src/goldToken/selectors.ts

```diff
@@ -0,0 +1,5 @@
+import { RootState } from 'src/redux/reducers'
+
+export function getCeloBalance(state: RootState) {
```

Comment:
Nit: matter of style preference, but most of our selectors follow the () => {} syntax